README.md

```diff
@@ -153,29 +211,37 @@ Now that you have created a machine learning model, you can utilize the OpenScal
 
 ### Sample Output
 
-* Go to the instance of [Watson OpenScale](https://aiopenscale.cloud.ibm.com/aiopenscale/insights) that you created and click `Manage` on the menu and then `Launch Application`. Choose the `Insights` tab to get an overview of your monitored deployments, Accuracy alerts, and Fairness alerts.
+You can find a sample notebook with output for [WatsonOpenScaleMachineLearning-with-outputs.ipynb](notebooks/with-output/WatsonOpenScaleMachineLearning-with-outputs.ipynb).
```

Comment:
Broken link: notebooks/with-outputs/WatsonOpenScaleMachineLearning-with-outputs.ipynb

Reply:
Yeah, that won't work until this PR is merged....it didn't exist previously._

Reply:
No I meant it should be notebooks/with-outputs and not notebooks/with-output

Reply:
oh, sorry. Fixed.